### PR TITLE
Add auto release on tag

### PR DESCRIPTION
**Pull request by Velenir**
This should make travis zip `./dist` folder and publish to releases when it sees a tag
Will see if it works when we make a new tag

### CODE REVIEW

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr324--dexreact.review.gnosisdev.com

File: .travis.yml [hunk 1]
```diff
@@ -11,6 +11,13 @@ before_install:
   # Needed to deploy pull request and releases
   - sudo apt-get -y install python-pip python-dev
   - pip install awscli --upgrade --user
+before_deploy:
+  # zip ./dist folder if tag IS present
+  - if [ $TRAVIS_TAG != "" ]; then
+      pushd dist;
+      zip -r  $OLDPWD/release_$TRAVIS_TAG.zip *;
+      popd;
```
Comment (alfetopito) [inline .travis.yml]: Was not familiar with pushd/popd. Is this essentially the same as `cd dist` then `cd -` ?

Comment (Velenir) [reply .travis.yml]: Yep, was recommended where I copied it from

File: .travis.yml [hunk 2]
```diff
@@ -50,3 +57,11 @@ deploy:
     script: bash travis/prepare_production_deployment.sh
     on:
       tags: true
+
+  # add release_v*.zip file to tagged Releases on github
+  - provider: releases
+    api_key: $GITHUB_GNOSIS_INFO_API_TOKEN
```
Comment (anxolin) [inline .travis.yml]: Don't you need to add this secret?

**Pull request merged**